charts/falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## Unreleased (0.9.4)
```

Comment:
```suggestion
## 0.9.4
```

You can directly set the version, the CI enforces to bump the version at each PR